src/_site/assets/i/icons/tip.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this file from the PR.

Reply:
It was mistakenly added to this PR. Sorry 😊